Classes/ARLocalTiledImageDataSource.m

```diff
@@ -38,7 +73,11 @@ - (NSInteger)minimumImageZoomLevelForImageView:(ARTiledImageView *)imageView
 
 - (NSInteger)maximumImageZoomLevelForImageView:(ARTiledImageView *)imageView
 {
-    return self.maxTileLevel;
+    return self.maxTileLevel + _extralVisualLevel;
```

Comment:
This means that the maximumZoomLevel has become +1 by default? I don't think the default behavior should change.